File: Multiples_of_3_and_5/solution.js
function solution(number){
    var sum = 0
    if (number < 0){
        return 0
    }  
    for (let i = 0; i <number; i++) {
        if (i % 3 & i % 5 == 0) {
            sum += i;
        }
        else if (i % 3 == 0) {
            sum += i;
        }
        else if (i % 5 == 0) {
            sum += i;
        }
    }
    return sum;
}

File: Multiples_of_3_and_5/solution.py
def solution(number):
    sum = 0
    if number < 0:
        return 0
    for i in range(0, number):
        if i % 3 == 0 and i % 5 == 0:
            sum += i
        elif i % 3 == 0:
            sum += i
        elif i % 5 == 0:
            sum += i
    return(sum)

